test cases/common/48 test args/copyfile.py

```diff
@@ -3,4 +3,15 @@
 import sys
 import shutil
 
+# If either of these use `\` as the path separator, it will cause problems with
```

Comment:
I would consider adding a TODO comment here saying something like
`It is okay to have \ as separator for non-mingw/msys Windows environment, however this test is more strict for simplicity`. In that case if later for whatever reason we decide to change it we will have less doubt.

Reply:
It's not okay for those either. People do use msys tools while building with msvc, so we cannot change it anywhere.

Reply:
Eventually, if people start reporting bugs about tools that require `\` path separators, we will have to think of some way to fix this. It's a hard problem.